var/spack/repos/builtin/packages/lorene/package.py

```diff
@@ -112,6 +112,8 @@ def build(self, spec, prefix):
     def install(self, spec, prefix):
         mkdirp(prefix.lib)
         install_tree("Lib", prefix.lib)
+        install_tree("Export/C++/Include", prefix.include)
```

Comment:
Is there a call to `mkdirp` missing?

Reply:
@eschnett I don't believe so. I have tested this script.

Reply:
This may depend on whether another package already created that `include` directory at `prefix`. It would be weird if the `include` directory was created automatically but the `lib` directory wasn't.